docs/source/build-instructions.rst

```diff
@@ -0,0 +1,44 @@
+Brave Build Instructions
```

Comment:
It's potentially confusing to duplicate documentation across the GitHub wiki and ReadTheDocs. I suggest you either:

1. Leave developer docs on the GitHub Wiki (for now, you can always do a follow-up PR to migrate them) and only use ReadTheDocs for end user docs.
2. Migrate all of the GitHub wiki docs to ReadTheDocs, and retire the GitHub wiki. I recommend cloning the GitHub wiki's underlying git repo, using Pandoc to convert GFM to RST, and skimming the converted docs to make sure there are no serious issues.

    If you decide to do this, note that there are still a lot of links to GitHub wiki pages floating around in Slack history, email threads, etc. It might be worthwhile to convert each page of the wiki into a redirect page, e.g. a page that just says "Documentation that was formerly hosted here has been migrated to https://brave-browser.readthedocs.io/<new-url>."